middleware/http/wasm/internal/testdata/rewrite.wat

```diff
@@ -0,0 +1,92 @@
+;; This example module is written in WebAssembly Text Format to show the
```

Comment:
Is it generated as well?

Reply:
no I wrote it. this is the baseline for benchmarks as compiled with tinygo etc won't be faster than this.

Reply:
I will change the comment!